integration-tests/features/steps/stack_analysis.py

```diff
@@ -812,6 +828,7 @@ def validate_topic_list(context, key):
     json_data = context.response.json()
     path = "result"
     manifest_results = get_value_using_path(json_data, path)
+    print(manifest_results)
```

Comment:
are these print statements needed?

Reply:
Not necessary I had just put them for debugging forgot to remove